hack/make-rules/test.sh

```diff
@@ -289,7 +289,7 @@ runTests() {
       ${KUBE_RACE} ${KUBE_TIMEOUT} "${@}" \
      "${testargs[@]:+${testargs[@]}}" \
      | tee ${junit_filename_prefix:+"${junit_filename_prefix}.stdout"} \
-     | grep "${go_test_grep_pattern}" && rc=$? || rc=$?
+     | grep --binary-files=text "${go_test_grep_pattern}" && rc=$? || rc=$?
```

Comment:
@sttts what does this do in context?

Reply:
it forces grep to consider output it might otherwise think is binary as text, so that it outputs the actual match instead of just saying "binary file matched".  Since it's just test output, we should never have *actual* binary junk, just stuff that grep incorrectly assumes is binary.

Reply:
I hit this when debugging a fuzzer failure -- it thought the fuzzer output was binary, and so it just truncated the results of the test output